tsconfig.json

```
@@ -0,0 +1,17 @@
+{
+  "extends": "@tsconfig/node20/tsconfig.json",
+  "include": [
+    "src/**/*"
+  ],
+  "exclude": [
+    "node_modules",
+    "**/*.spec.ts"
+  ],
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "outDir": "lib",
+    "declaration": true,
+    "declarationMap": true,
+    "lib": ["ES2023"]
```

If we add cypress types here, we can finally remove [this import](https://github.com/saucelabs/cypress-junit-plugin/pull/1/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R1) and the [globals from eslint](https://github.com/saucelabs/cypress-junit-plugin/pull/1/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R9).
```suggestion
    "lib": ["ES2023"],
    "types": ["cypress"]
```